tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3057,3 +3057,70 @@ def crop_and_resize_v2(
     None, 'box_ind is deprecated, use box_indices instead', 'box_ind')
 tf_export(v1=['image.crop_and_resize'])(
     crop_and_resize_deprecation(gen_image_ops.crop_and_resize))
+
+@tf_export(v1=['image.draw_bounding_boxes'])
+def draw_bounding_boxes(
+    images,
+    boxes,
+    name=None):
+  """Draw bounding boxes on a batch of images.
+
+  Outputs a copy of `images` but draws on top of the pixels zero or more bounding
+  boxes specified by the locations in `boxes`. The coordinates of the each
+  bounding box in `boxes` are encoded as `[y_min, x_min, y_max, x_max]`. The
+  bounding box coordinates are floats in `[0.0, 1.0]` relative to the width and
+  height of the underlying image.
+
+  For example, if an image is 100 x 200 pixels (height x width) and the bounding
+  box is `[0.1, 0.2, 0.5, 0.9]`, the upper-left and bottom-right coordinates of
+  the bounding box will be `(40, 10)` to `(180, 50)` (in (x,y) coordinates).
+
+  Parts of the bounding box may fall outside the image.
+
+  Args:
+    images: A `Tensor`. Must be one of the following types: `float32`, `half`.
+      4-D with shape `[batch, height, width, depth]`. A batch of images.
+    boxes: A `Tensor` of type `float32`.
+      3-D with shape `[batch, num_bounding_boxes, 4]` containing bounding
+      boxes.
+    name: A name for the operation (optional).
+
+  Returns:
+    A `Tensor`. Has the same type as `images`.
+  """
+  return gen_image_ops.draw_bounding_boxes(images, boxes, name)
+
+@tf_export('image.draw_bounding_boxes', v1=[])
+def draw_bounding_boxes_v2(
```

Comment:
Might as well keep the colors functionality in both V1 and V2, no?
Something like:

```python
@tf_export('image.draw_bounding_boxes', v1=[])
def draw_bounding_boxes_v2(images, boxes, colors=None, name=None):
  if colors is None and not compat.forward_compatible(2019, 05, 01):
    return gen_image_ops.draw_bounding_boxes(images, boxes, name)
  return gen_image_ops.draw_bounding_boxes_v2(images, boxes, colors, name)

@tf_export(v1=['image.draw_bounding_boxes'])
def draw_bounding_boxes(images, boxes, name=None, colors=None):
  return draw_bounding_boxes_v2(images, boxes, colors, name)
```

We don't need separate wrappers for each of the generated functions.
Thoughts?

Where `compat.forward_compatible` is from: https://github.com/tensorflow/tensorflow/blob/2f2503d5d0c27104848fcee418ca9f97607cc39c/tensorflow/python/compat/compat.py#L34

Reply:
Updated.